pyensembl/download_cache.py

```diff
@@ -12,14 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from os import listdir, remove
+from os import listdir, remove, environ
 from os.path import join, exists, split, abspath
 from shutil import copy2, rmtree
 import logging
 
 import datacache
 
 CACHE_BASE_SUBDIR = "pyensembl"
+ENV_KEY = "PYENSEMBL_CACHE_DIR"
```

Comment:
nit: might call this something a bit clearer and non-abbreviated like `CACHE_DIRECTORY_ENVIRONMENT_VARIABLE`